lib/babe/verify.go

```diff
@@ -409,15 +413,28 @@ func (b *verifier) verifyPreRuntimeDigest(digest *types.PreRuntimeDigest) (scale
 	case types.BabePrimaryPreDigest:
 		ok, err = b.verifyPrimarySlotWinner(d.AuthorityIndex, d.SlotNumber, d.VRFOutput, d.VRFProof)
 	case types.BabeSecondaryVRFPreDigest:
+		if !b.secondarySlots {
+			ok = false
+			break
+		}
 		pub := b.authorities[d.AuthorityIndex].Key
-		var pk *sr25519.PublicKey
-		pk, err = sr25519.NewPublicKey(pub.Encode())
+
+		pk, err := sr25519.NewPublicKey(pub.Encode()) // nolint
+		if err != nil {
+			return nil, err
+		}
+
+		ok, err = verifySecondarySlotVRF(&d, pk, b.epoch, len(b.authorities), b.randomness) // nolint
```

Comment:
Same here, let's not use `// nolint` anymore. You can specify the linter you want to be ignored with `//nolint:lintername`. Note there is no space between `//` and `nolint`.

Reply:
I removed //nolint tag completely, and linter no longer complains, has something changed?

Reply:
Well yes, `golangci-lint` version wasn't pinned in the CI, so maybe that was a false positive back months ago and now it doesn't detect it anymore. We should remove these `// nolint` comments sometime soon since they could be hiding potentially bad code too.